### PR TITLE
Add AC Version to version info page

### PR DESCRIPTION
**Pull request by kaxil**
image

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by ashb** [comment]
We should probably remove the local part from the airflow version, as that links out to the project on pypi (which would 404 for the astro version)

File: airflow/www_rbac/templates/airflow/version.html [hunk 1]
```diff
@@ -24,11 +24,15 @@
...
-  {% set version_label = 'Version' %}
     {% if airflow_version %}
-        <h4>{{ version_label }} : <a href="https://pypi.python.org/pypi/apache-airflow/{{ airflow_version }}">{{ airflow_version }}</a></h4>
+        <h4>Airflow Version : <a href="https://pip.astronomer.io/simple/apache-airflow/">{{ airflow_version }}</a></h4>
     {% else %}
-        <h4>{{ version_label }} : Not Available</h4>
+        <h4>Airflow Version : Not Available</h4>
+    {% endif %}
+    {% if ac_version %}
+        <h4>AC Version : <a href="https://pip.astronomer.io/simple/astronomer-certified/">{{ ac_version }}</a></h4>
```
Comment (ashb) [inline airflow/www_rbac/templates/airflow/version.html:33]: This isn't a great place to link to, it's just a unstyled download page. I think for now lets have this not be a link.

**Comment by kaxil** [comment]
> We should probably remove the local part from the airflow version, as that links out to the project on pypi (which would 404 for the astro version)

Do we want to link out to Airflow version on PyPI ?? as it won't be the same as the version showed.

**Comment by kaxil** [comment]
Changed to the following:

Apr-16-2020 15-38-16

**Comment by kaxil** [comment]
**Dev Version**:
Apr-16-2020 20-38-09

**Stable Version**:
Apr-16-2020 21-01-15

File: airflow/www_rbac/templates/airflow/version.html [hunk 2]
```diff
@@ -24,13 +24,27 @@
 {% block content %}
     {{ super() }}
     <h2>{{ title }}</h2>
-  {% set version_label = 'Version' %}
     {% if airflow_version %}
-        <h4>{{ version_label }} : <a href="https://pypi.python.org/pypi/apache-airflow/{{ airflow_version }}">{{ airflow_version }}</a></h4>
+      {% set airflow_non_local_version = airflow_version.split('.dev')[0].split('+astro')[0] %}
```
Comment (ashb) [inline airflow/www_rbac/templates/airflow/version.html:28]: This is probably complex enough it should be in the view, not in the template -- let's just feed the template the right values it needs.

Comment (ashb) [reply airflow/www_rbac/templates/airflow/version.html:28]: And lets call this variable "airflow_upstream_version"

Comment (kaxil) [reply airflow/www_rbac/templates/airflow/version.html]: Updated

**Pull request merged**